.github/workflow/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] The folder containing this file is supposed to be named `.github/workflows`, but is currently named `workflow` (it's missing the last **s**). Please rename the folder to activate GitHub actions on your Pull Request. You should be able to see a message like this at the bottom of your Pull Request once you do this: 
![image](https://user-images.githubusercontent.com/2739245/125147212-80f0c380-e0ef-11eb-800a-cf18731dcfa0.png)

- [ ] Once you see this, please make sure to click on the `Details` link for any failed linter tests you see here, and fix them before submitting another CRR.